systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -288,14 +290,16 @@ def post_configure(self, address):
         """ The steps that must be done after a device is configured """
         route = CsRoute()
         if not self.get_type() in ["control"]:
-            route.add_table(self.dev)
 
-            CsRule(self.dev).addMark()
+            if self.dev != 'eth0':
+                route.add_table(self.dev)
+                CsRule(self.dev).addMark()
+                self.set_mark()
 
             interfaces = [CsInterface(address, self.config)]
             CsHelper.reconfigure_interfaces(self.cl, interfaces)
 
```

Comment:
Lines 299 and 300 were added as part of c41edc1. I've not investigated what those lines do, I've just naively overcome the merge conflicts. I'll look at the results of CI and follow from there.